bigquery/tables.js

```diff
@@ -15,308 +15,412 @@
+  // const projectId = "your-project-id";
+
+  // The ID of the dataset of the table into which data should be imported, e.g. "my_dataset"
+  // const datasetId = "my_dataset";
+
+  // The ID of the table into which data should be imported, e.g. "my_table"
+  // const tableId = "my_table";
+
+  // The name of the Google Cloud Storage bucket where the file is located, e.g. "my-bucket"
+  // const bucketName = "my-bucket";
+
+  // The name of the file from which data should be imported, e.g. "file.csv"
+  // const filename = "file.csv";
+
   // Instantiates clients
   const bigquery = BigQuery({
     projectId: projectId
   });
+
   const storage = Storage({
     projectId: projectId
   });
 
-  // References an existing dataset, e.g. "my_dataset"
-  const dataset = bigquery.dataset(datasetId);
-  // References an existing table, e.g. "my_table"
-  const table = dataset.table(tableId);
-  // References an existing bucket, e.g. "my-bucket"
-  const bucket = storage.bucket(bucketName);
-  // References an existing file, e.g. "file.txt"
-  const file = bucket.file(fileName);
-
   let job;
 
-  // Exports data in a table into a Google Cloud Storage file
-  return table.export(file)
+  // Exports data from the table into a Google Cloud Storage file
+  bigquery
+    .dataset(datasetId)
+    .table(tableId)
+    .export(storage.bucket(bucketName).file(filename))
     .then((results) => {
       job = results[0];
       console.log(`Job ${job.id} started.`);
       return job.promise();
     })
     .then((results) => {
       console.log(`Job ${job.id} completed.`);
-      return results;
+    })
+    .catch((err) => {
+      console.error('ERROR:', err);
     });
+  // [END bigquery_export_gcs]
 }
-// [END bigquery_export_gcs]
 
-// [START bigquery_insert_stream]
 function insertRowsAsStream (datasetId, tableId, rows, projectId) {
+  // [START bigquery_insert_stream]
+  // Imports the Google Cloud client library
+  const BigQuery = require('@google-cloud/bigquery');
+
+  // The project ID to use, e.g. "your-project-id"
+  // const projectId = "your-project-id";
+
+  // The ID of the dataset of the table into which data should be inserted, e.g. "my_dataset"
+  // const datasetId = "my_dataset";
+
+  // The ID of the table into which data should be inserted, e.g. "my_table"
+  // const tableId = "my_table";
+
   // Instantiates a client
   const bigquery = BigQuery({
     projectId: projectId
   });
 
-  // References an existing dataset, e.g. "my_dataset"
-  const dataset = bigquery.dataset(datasetId);
-  // References an existing table, e.g. "my_table"
-  const table = dataset.table(tableId);
-
   // Inserts data into a table
-  return table.insert(rows)
+  bigquery
+    .dataset(datasetId)
+    .table(tableId)
+    .insert(rows)
     .then((insertErrors) => {
       console.log('Inserted:');
       rows.forEach((row) => console.log(row));
-      return insertErrors;
+
+      if (insertErrors && insertErrors.length) {
```

Comment:
Why are we checking `insertErrors.length` - if we want this to be `insertErrors.length > 0`, can we be explicit about that?